src/plat/qemu-arm-virt/config.cmake

```diff
@@ -107,7 +107,7 @@ if(KernelPlatformQEMUArmVirt)
                 string(
                     REGEX
                         MATCH
-                        "[0-9](\\.[0-9])+"
+                        "[0-9]+(\\.[0-9]+)+"
```

Comment:
The old regex would capture M.N version out of M.N.X, your new ones captures the full M.N.X. which is more than needed to handle qemu 10.n and isn't explained why it was done. `MIN_QEMU_VERSION` that we check against is "3.1.0", so this change is wanted, even though `VERSION_LESS` probably works either way. But please document all changes and the reason for them in the commit message.

Reply:
Fair. It was a drive-by fix, and it seemed more strange to support XX.Y.Z but not XX.YY.ZZ. I'm happy to revert the secondary change or edit the commit message.

Reply:
Never mind, I misread the diff,  I thought you added the third `+` instead of the second one for some reason.